apps/api/v2/src/modules/atoms/services/event-types-atom.service.ts

```diff
@@ -77,4 +101,120 @@ export class EventTypesAtomService {
 
     return eventType.eventType;
   }
+
+  async getEventTypesAppIntegration(slug: string, userId: number, userName: string | null, teamId?: number) {
```

Comment:
Will simplify this once #16946 merges so that we don't have to repeat this twice in two different place